src/app/content/components/Attribution.spec.tsx

```diff
@@ -177,5 +177,21 @@ describe('Attribution', () => {
 
       expect(details.children[1].innerHTML).toMatch(`Authors: Jhon Doe, Jonny Doe`);
     });
+
+    it('renders special licensing and attribution for HS Physics', async() => {
+      store.dispatch(
+        actions.receiveBook({...formatBookData(book, mockCmsBook), id: 'cce64fde-f448-43b8-ae88-27705cceb0da'})
+      );
+      const { node: details } = renderToDom(render());
+      details.setAttribute('open', '');
```

Comment:
This doesn't actually make a difference for this test case, but i don't think there is a way to assert on the text being visible (instead of just being in the document)

Reply:
yeah I think it being in the document is enough